<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comandU</class>
 <widget class="QWidget" name="comandU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>691</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="twComandU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="teComandU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leComandNameSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string extracomment="Введите ФИО Ученика для поиска"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>На главную</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Окно отладки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectAll">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать все отделы</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Отделы</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
